outline.md

```diff
@@ -0,0 +1,57 @@
+## Growing an Inner Source Program
+
+45 min
+
+## Short Description
+
+"Inner source is a great idea! Go do inner source!"
+You're the dedicated champion for inner source in your company.
+What do you do now?
+How do you realistically affect the behavior of dozens or hundreds of teams to the point where robust and pervasive inner sourcing is a normal part of the way that engineering is done?
+
+This presentation shares principles, practical experience, and relatable examples on this situation.
+While not claiming all of the answers, it is insightful summary of over a year experience and work in the area.
+
+## Detailed description
+
+#### Outline
+
+An inner source program is the set of tools, documentation, training, and so forth provided so that any team can participate in inner source effectively as either a maintainer or contributor.
+To frame the talk, we'll explain the goals of an inner source program as well as the context within-which we've been pursuing those goals.
+Even though the learning in this presentation occurred in that context, we suspect that much of what's here applies generally.
+
+There are two broad areas of planning in building an inner source program.
+The first area is the vision of what services you will provide to facilitate inner sourcing.
+The second is what strategy you will employ to roll out and make effective these services in your organization.
+We'll explain principles, resources, and specific examples in each area.
+
+In the first area, the services of the program should provide two things.
+First, they must teach, define, and document the type of culture that is conducive to inner source.
+Second, there must be software that simultaneously reinforces that culture while also lowering the barrier to entry to participating in that culture.
+Examples of these services include:
+
+* Documentation on how to do inner source (and relation of that documentation to [InnerSourcePatterns](http://bit.ly/2EoEH4f)).
```

Comment:
for the link, you might give it as: [InnerSourcePatterns](http://innersourcecommons.org/patterns)

Reply:
Sure!